lib/Doctrine/DBAL/Schema/OracleSchemaManager.php

```diff
@@ -36,6 +36,65 @@ class OracleSchemaManager extends AbstractSchemaManager
     /**
      * {@inheritdoc}
      */
+    public function listTables()
+    {
+        $currentDatabase = $this->_conn->getDatabase();
+
+        $tableNames = $this->listTableNames();
+
+        // Get all column definitions in one database call.
+        $columnsByTable = $this->getAssetRecordsByTable($this->_platform->getListColumnsSQL($currentDatabase, null));
+
+        // Get all foreign keys definitions in one database call.
+        $foreignKeysByTable = $this->getAssetRecordsByTable($this->_platform->getListForeignKeysSQL($currentDatabase, null));
+
+        // Get all indexes definitions in one database call.
+        $indexesByTable = $this->getAssetRecordsByTable($this->_platform->getListIndexesSQL($currentDatabase, null));
+
+        $tables = [];
+        foreach ($tableNames as $tableName) {
+            $unquotedTableName = trim($tableName, '"');
```

Comment:
The quote should be replaced with `$this->_platform->getIdentifierQuoteCharacter()`. It's not an unquoted table name, it's just a table name. And the original `$tableName` is the quoted table name.